hybrasyl/ControlMessages/ControlMessage_Spawn.cs

```diff
@@ -20,8 +20,10 @@ private void ControlMessage_MonolithControl(HybrasylControlMessage message)
             var monster = (Monster)message.Arguments[0];
             var map = (Map)message.Arguments[1];
 
-            // Don't handle control messages for dead/removed mobs
-            if (!monster.Condition.Alive || monster.DeathProcessed || monster.Id == 0 || monster.Map == null) return;
+            // Don't handle control messages for dead/removed mobs, or mobs that cannot move or attack
+            if (!monster.Condition.Alive || monster.DeathProcessed || 
```

Comment:
how is condition blind handled? they cannot walk, but can still attack

Reply:
blind is no walk, no spells

Reply:
yeah, but how is it being handled for control messages

Reply:
fixed, blind will work similar to usda now